src/material/sort/sort-header.ts

```diff
@@ -127,7 +127,16 @@ export class MatSortHeader extends _MatSortHeaderBase
   @Input('mat-sort-header') id: string;
 
   /** Sets the position of the arrow that displays when sorted. */
-  @Input() arrowPosition: 'before' | 'after' = 'after';
+  @Input()
+  get arrowPosition(): 'before' | 'after' { return this._arrowPosition; }
+  set arrowPosition(v) {
```

Comment:
I'm not sure that we need the getter/setter here since the default is already being set in the constructor. The types will ensure that the correct value is passed in.

Reply:
Done.